tests/system/providers/yandex/example_yandexcloud_dataproc.py

```diff
@@ -153,3 +156,8 @@
 
     create_cluster >> create_mapreduce_job >> create_hive_query >> create_hive_query_from_file
     create_hive_query_from_file >> create_spark_job >> create_pyspark_job >> delete_cluster
```

Comment:
Shouldn't the ``delete_clusted`` be with ``Trigger_Rule.ALL_DONE``? (and then use watcher task to watch the status of the DAG). In case there is any failure in the DAG, we're left with clusted created by ``create_cluster`` task.

Reply:
Thanks @bhirsz , i will push a commit to fix this problem.